crates/hdk/src/link.rs

```diff
@@ -122,8 +122,8 @@ pub fn delete_link(address: ActionHash, get_options: GetOptions) -> ExternResult
     })
 }
 
-/// Returns all links that reference a base hash, filtered by link type and other criteria.
-/// Use a [`GetLinksInputBuilder`] to create the [`GetLinksInput`] and optionally filter links further.
+/// Returns all links that reference a base hash, filtered by the provided [`LinkQuery`] to apply diffent filters
+/// and the [`GetStrategy`], which can be used to specify whether to look up the links locally or fetch them from the network.
```

Comment:
Technically what we do is retrieve links from the network to update our local set and then query locally. So perhaps end this bit with something like "look up the links locally or fetch update our links by fetching links from the network"

Not very happy with my wording, but something along those lines would be a little more precise.